### ISSUE TITLE
Problem with File Save/Open

### ISSUE THREAD

**Issue by ElbyMann** [notahat/midi_patchbay#23]
I've downloaded/built on Sonoma 14.7.1. The app works as expected, except that the File functions - Open/Save/Save as don't do anything - no window opens when these are selected.  Happy to help troubleshoot this.

**Comment by ben-juodvalkis** [comment]
Did you ever figure this out?

**Comment by ElbyMann** [comment]
Nope.  i ended up writing python scripts to do my midi routing, which works great!

**Comment by MultifokalHirn** [comment]
Adressed in https://github.com/notahat/midi_patchbay/pull/22
Maintainer is sadly not merging and appears to be inactive.
